Modelica/Utilities/Internal.mo

```diff
@@ -240,4 +240,233 @@ especially if the operating system supports Unicode characters.
 </p>
 </html>"));
 end FileSystem;
+
+  package Time "Internal package with external functions as interface to date and time"
+    extends Modelica.Icons.InternalPackage;
+
+    pure function diffTime "Convert local time to elapsed seconds since custom epoch year"
+      extends Modelica.Icons.Function;
+      input Integer ms "Millisecond";
+      input Integer sec "Second";
+      input Integer min "Minute";
+      input Integer hour "Hour";
+      input Integer day "Day";
+      input Integer mon "Month";
+      input Integer year "Year";
+      input Integer epoch_year = 1970 "Reference year";
+      output Real seconds "Elapsed seconds since epoch_year in the current time zone";
```

Comment:
Shouldn't it be since _start of_ epoch_year?

Reply:
OK.